module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -204,6 +204,19 @@ public function supportsRecovery($authMethod = null)
             && $this->getAuth($authMethod)->supportsPasswordRecovery();
     }
 
+    /**
+     * Multi-ILS Only, Does the target Auth Method support password Recovery?
+     *
+     * @param $target string the target to check against
+     *
+     * @return bool
+     */
+    public function multiSupportsPasswordRecovery($target)
```

Comment:
Might it be possible to add a second optional argument to supportsPasswordRecovery instead of a whole new method here? Or is this really unavoidable?